src/ContextMenu.tsx

```diff
@@ -70,13 +72,41 @@ const ContextMenuView: React.FunctionComponent<IContextMenuView> = props => {
   const { menuItems = [], ...viewProps } = props;
 
   const { contextMenu: theme } = useTheme();
+  const [isVisible, setIsVisible] = React.useState(false);
 
   return (
-    <Box {...viewProps} as={ReactContextMenu} css={menuStyles(theme, menuItems.length > 0)}>
-      {menuItems.map((item, index) => {
-        return <ContextMenuItem key={index} {...item} />;
-      })}
-    </Box>
+    <>
+      {props.blockExternalClicks && isVisible
+        ? ReactDOM.createPortal(
+            <Box
+              onClick={e => {
+                e.stopPropagation();
+                setIsVisible(false);
+              }}
+              width="100vw"
+              height="100vh"
+              position="absolute"
+              zIndex={10000}
```

Comment:
Shall we increase it?
I'd say we should follow the Overlay approach - 2^31 and some offset.
```suggestion
              zIndex={2 ** 31 - 3}
```